phpstan.neon

```diff
@@ -33,10 +33,6 @@ parameters:
     ignoreErrors:
         - '#Method App\\Form\\ProjectFormType\:\:buildForm\(\) has parameter \$formBuilder with no value type specified in iterable type Symfony\\Component\\Form\\FormBuilderInterface#'
 
-       # no idea how to fix these now, possible sources: https://github.com/phpstan/phpstan-doctrine/blob/e560d32f6fb68e5bbec0e556a69723e7faece14a/tests/Rules/Doctrine/ORM/data/EntityWithRelations.php
-        - '#Method (.*?) return type with generic interface Doctrine\\Common\\Collections\\Collection does not specify its types\: TKey, T#'
-        - '#Property (.*?) with generic interface Doctrine\\Common\\Collections\\Collection does not specify its types\: TKey, T#'
```

Comment:
@Kerrialn FIY, I took a Sunday off and made rule for this one.

https://github.com/rectorphp/rector/pull/4442#issue-505480715

It's part of `SetList::DOCTRINE_CODE_QUALITY` set